lib/update_remote_id.rb

```diff
@@ -0,0 +1,51 @@
+# Update events remote_id from existing schedule xml data
```

Comment:
This is called from the API controller:

https://github.com/voc/voctoweb/blob/63317953452fcdddbcfc36bddb40ef4db2f5df5a/app/models/concerns/fahrplan_updater.rb#L17

However I should add the metadata to https://github.com/voc/voctoweb/blob/431ed8efd86d6403fa8aa99c078438b2d383e8f6/app/controllers/api/events_controller.rb#L93 so you can set it directly via the API.